corehq/apps/locations/dbaccessors.py

```diff
@@ -40,3 +44,25 @@ def get_all_users_by_location(domain, location_id):
         reduce=False,
     )
     return (CouchUser.wrap_correctly(res['doc']) for res in results)
+
+
+def users_have_locations(domain):
+    from corehq.apps.users.models import CouchUser
+    return bool(CouchUser.get_db().view(
+        'users_extra/users_by_location_id',
+        startkey=[domain],
+        endkey=[domain, {}],
+        reduce=True,
```

Comment:
can we add `stale=True` to this? (and possibly move to `analytics` or at least rename to indicate that its a stale query)? didn't see any calls to this that needed up-to-the-second data